src/main/java/com/crud/library/domain/Rental.java

```diff
@@ -0,0 +1,25 @@
+package com.crud.library.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+@Entity(name = "RENTALS")
+public class Rental {
+
+    @Id
+    @GeneratedValue
+    private long id;
+    private LocalDate rentDate;
+    private LocalDate returnDate;
+    @OneToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
czemu @OneToOne?

Reply:
dlatego, ze wypożyczył konkretny egzemplarz, można stworzyć kolejny egzemplarz na podstawie book i ten sobie wypożyczy jeszcze kto inny; danego egzemplarza nie może wypożyczyć wiele osób, ale dana książkę już tak

Reply:
Ale .... idziesz do biblioteki i masz tam konkretna liczbe egzemplarzy danej ksiazki. Wiecej niz jest nie wypozyczysz - nie tworzysz nowego egzemplarza.
A @OneToOne nie pozwoli na wpis wypozyczenia egzemplarza z id=1 dwom uzytkownikow
W tabelce bedziemy mogli przeciez miec takie dwa wiersze:
37, 13-03-2021, 21-03-2021, 1, 5
98, 24-08-2021, 14-09-2021, 1, 1554
U ciebie to niemozliwe bedzie, bo OneToOne mowi, ze tylko raz moze wystapic ksiazka z id=1 w tej tabelce